### ISSUE TITLE
Cannot use the numpad7 key to go up line by line in the reader

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4050]
**Reported by vgjh2005 on 2014-04-04 07:06**
Hi
My system is windows 8 32bit.I like to use the reader to read PDF document file. I fine a problem by using NVDA to read the book. I used the numpad9 to read the content line by line. But when I press numpad7 to read the last line, NVDA would move the review cursor to the top.
Thanks.

**Comment by bhavyashah** [comment]
@k_kolev1985  @vgjh2005 Could you please clarify whether the reported issue is specific to the Reader app in Windows 8.1, or if the same buggy behaviour can be successfully reproduced elsewhere too?

**Comment by k-kolev1985** [comment]
Now I've tested it with the Reader app in Windows 10 and the reading line by line seams to be completely broken. I can provide an NVDA log containing the error if needs be, but I think it should be in a new ticket. What do you say?

**Comment by ehollig** [comment]
@k-kolev1985, it may be best if we have updated steps to reproduce and a log file provided in a new ticket. I will close this for now, but if this still persists using Windows 8.1, we can reopen this issue.
